pkg/agent/discoverycache.go

```diff
@@ -20,7 +22,7 @@ import (
 const (
 	apiServerPageSize   = 100
 	healthcheckEndpoint = "central"
-	apiServerFields     = "metadata,name,title,owner,attributes"
+	apiServerFields     = "metadata,name,title,owner,attributes,x-agent-details"
```

Comment:
do we still need/want attributes?

Reply:
We still need it for consumer instances. Consumer instances save all the attributes, and also save the same attributes in the x-agent-details. Once we can stop querying for consumer instances on the api id, then we can probably remove the attributes.